### ISSUE TITLE
Display alt text in image preview

### ISSUE THREAD

**Issue by trwnh** [tootsuite/mastodon#7859]
Currently, alt text is only shown when hovering over the image in columns, or when the image fails to load. I propose that the alt text also be shown as a caption when clicking on the image to load the lightbox.

* * * *
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
Semi-related: #6401, but that's for extending the current tooltip behavior to sensitive images, whereas this issue is for the lightbox.

**Comment by rkingett** [comment]
I agree. I am kind of amazed this isn't a thing already. But yes, this would be good.

**Comment by futzle** [comment]
An issue #33804 used to exist, but was deleted when the GitHub user deleted her account, so I'll repeat what I said there:

This is even worse since Mastodon 4.4, which removed hover-to-view alt text. There's now no way to simultaneously:

1. View an image full-screen (in the lightbox), and
2. See the alt text for an image.

The alt text commonly draws your attention to one specific part of an image which might not even be visible in the crop shown in the timeline. I've got to read the text, remember what it said, and then click on the image to show it in the lightbox.

A caption for each image corresponding to its alt text would be a satisfactory solution.
